dev/docker_image/Dockerfile

```
@@ -0,0 +1,123 @@
+
+# Install Picard 2.20.7
+RUN wget https://github.com/broadinstitute/picard/releases/download/2.20.7/picard.jar && chmod +x picard.jar
+
+# Install sambamba 0.6.6
+RUN wget https://github.com/lomereiter/sambamba/releases/download/v0.6.6/sambamba_v0.6.6_linux.tar.bz2 && tar -xvjf sambamba_v0.6.6_linux.tar.bz2 && mv sambamba_v0.6.6 sambamba && rm -rf sambamba_*
+
+# Install gsl 1.16
+RUN wget http://gnu.mirror.vexxhost.com/gsl/gsl-1.16.tar.gz && tar -zxvf gsl-1.16.tar.gz && cd gsl-1.16 && ./configure && make && make install && cd .. && rm -rf gsl-1.16 gsl-1.16.tar.gz
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
+
+# Install preseq 2.0.3
+RUN git clone --branch v2.0.3 --single-branch --recursive https://github.com/smithlabcode/preseq preseq_2.0.3 && cd preseq_2.0.3 && make && cd ../ && mv preseq_2.0.3/preseq . && rm -rf preseq_2.0.3
+
+# Install Bowtie2 2.3.4.3
+RUN wget https://github.com/BenLangmead/bowtie2/releases/download/v2.3.4.3/bowtie2-2.3.4.3-linux-x86_64.zip && \
+    unzip bowtie2-2.3.4.3-linux-x86_64.zip && mv bowtie2*/bowtie2* . && rm -rf bowtie2-2.3.4.3*
+
+# Install Bwa 0.7.17
+RUN git clone --branch v0.7.17 --single-branch https://github.com/lh3/bwa.git && \
+    cd bwa && make && cp bwa /usr/local/bin/ && cd ../ && rm -rf bwa*
+
+# Install phantompeakqualtools 1.2.1
+RUN wget https://github.com/kundajelab/phantompeakqualtools/archive/1.2.1.tar.gz && tar -xvf 1.2.1.tar.gz && rm -f 1.2.1.tar.gz
+ENV PATH="/software/phantompeakqualtools-1.2.1:${PATH}"
+
+# Install SAMstats
+RUN pip3 install --no-cache-dir SAMstats==0.2.1
+
+# Install IDR 2.0.4.2
+RUN git clone --branch 2.0.4.2 --single-branch https://github.com/kundajelab/idr && \
+    cd idr && python3 setup.py install && cd ../ && rm -rf idr*
+
+# Install system/math python packages (python2)
+RUN pip2 install --no-cache-dir numpy matplotlib==2.2.4
+
+# Install genomic python packages (python2)
+RUN pip2 install --no-cache-dir macs2==2.1.2.1 metaseq==0.5.6
+
+# Install UCSC tools (v377)
+RUN git clone https://github.com/ENCODE-DCC/kentUtils_bin_v377
+ENV PATH=${PATH}:/software/kentUtils_bin_v377/bin
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/software/kentUtils_bin_v377/lib
+
+# Prevent conflict with locally installed python outside of singularity container
+ENV PYTHONNOUSERSITE=True
+
+# Disable multithreading for BLAS
+ENV OPENBLAS_NUM_THREADS=1
+ENV MKL_NUM_THREADS=1
+
+# Get ENCODE chip-seq-pipeline container repository
+# This COPY assumes the build context is the root of the chip-seq-pipeline repo
+# and it gets whatever is checked out plus local modifications
+# so the buildling command should:
+# cd [GIT_REPO_DIR] && docker build -f dev/docker_image/Dockerfile .
+RUN mkdir -p chip-seq-pipeline/src
+COPY src chip-seq-pipeline/src/
+COPY chip.wdl chip-seq-pipeline/
+ENV PATH="/software/chip-seq-pipeline:/software/chip-seq-pipeline/src:${PATH}"
+
+# make some temporary directories
+RUN for i in $(seq 0 9); do mkdir -p /mnt/ext_$i; done
```

Here is an easier way to do this:
```suggestion
mkdir -p /mnt/ext_{0..9}; done
```

---

thanks. done.